addons/web/static/src/views/form/form_controller.scss

```diff
@@ -471,6 +471,20 @@
                 width: auto;
             }
         }
+
+        .o_field_widget {
+            &.o_field_many2one_avatar {
+                display: block;
+            }
```

Comment:
Hello @pparidans 
Yes I know that the actual difference is precisely between the `.o_m2o_avatar > img` and `.o_m2o_avatar > .o_m2o_avatar_empty` display value and to make both block I have given the css `display: block;` over here.
I have not given in `fields.scss` because of the possible side effects when the field is used in different context (like in list view, kanban view for example). For that purpose I have given the css in `form_controller.scss`.

So should I directly give the css `display: block;` in `fields.scss` ?? Kindly give me your suggestions.
Thanks!

Reply:
Fair enough about scoping it to the FormView... But your selector should target the `img` anyway.